README.md

```diff
@@ -54,7 +54,7 @@ After inclusion of the `AWSCommon.puml` file, there are two different ways to re
 
 1. **Use individual include files** - Use one file per service or setting. For example:
 
-    `!incude AWSPuml/Storage/AmazonSimpleStorageServiceS3.puml`
+    `!incude AWSPuml/Storage/S3Bucket.puml`
```

Comment:
`S3.puml` is no longer exists, so I use this one.

Reply:
Hi,

Thanks for noting the error. I've included in the latest README a correction to that line:

```
!incudeurl AWSPuml/Storage/SimpleStorageServiceS3.puml
```

to:

```
!includeurl AWSPuml/Storage/SimpleStorageServiceS3.puml
```

There was a missing `l` in `include`. I've left the individual object as `SimpleStorageServiceS3.puml` for continuity.